### PR TITLE
Body & Footer Fixes/Styling

### PR DESCRIPTION
**Pull request by edbarton-copart**
- added ability to style headers
- revised pagination logic
- fixed styling and cleaned up code
- added left/right page change
- removed border and added alternating row colors for cleaner look
- changed to MUI buttons/popover instead of CSS
- @jacobsiebert-copart added mouse events

### CODE REVIEW

**Comment by edbarton-copart** [comment]
Here is an example of the look of the table. Most of the changes were internal logic and style code cleanup. The look is largely unchanged except for the row colors, hover color, and selected row color.
image

**Comment by edbarton-copart** [comment]
Here is what it looks like with the changes.
image

File: src/RowSelectTable.css [hunk 1]
```diff
@@ -1,44 +1,40 @@
 .table {
   width: 100%;
-
   font-size: 12px;
 }
-
+tr:nth-child(even) {
```
Comment (anilreddykatta) [inline src/RowSelectTable.css]: @edbarton-copart Possible to take these kind of colors as prop too? We can have value defaulted to this value?

File: src/RowSelectTable.js [hunk 2]
```diff
@@ -4,48 +4,60 @@ import cn from 'classnames'
...
+    {columnMetadata.map(({ headerCellContent, name, sortable }) =>
+      <th key={name} name={name} className={cn(headerCell, { [sortableColumnHeaderCell]: sortable })} style={headerStyle} onClick={() => sortable && changeSort(name)}>
+        {(headerCellContent || headerCellContent === '') ? headerCellContent : name}
+        {sortable && sortColumnName === name && <RowSelectTableSortIcon direction={sortDirection} />}
+      </th>
+    )}
+  </tr>
   </thead>
 
 export const RowSelectTableRow = ({
```
Comment (anilreddykatta) [inline src/RowSelectTable.js]: Can we integrate prettier for this project ? @Shobhit1

**Comment by Shobhit1** [comment]
I guess this can be abandoned now! @tenzincopart

File: src/RowSelectTable.js (hunk 2 above)
Comment (colshacol) [reply src/RowSelectTable.js]: Good idea. 👍

**Comment by colshacol** [comment]
1 year later... Closing.

Let me know if we need to re-open for any reason.

**Pull request closed without merging**